js/directives/day-block.js

```diff
@@ -15,7 +15,7 @@
 			'</div>'+
 
 			'<div ng-style="controller.monthStyleOverride()" ' +
-			'class="month">{{controller.date | luifMoment: \'MMM\' | limitTo : 3}}'+
+			'class="month">{{controller.date | luifMoment: \'MMM\' | fancyMonth}}'+
```

Comment:
Because the french MMM format could returns until 5 characters like in "Janv." or "Juil." I think that @Clotilde wanted 3 characters only. Personally I prefer this:

`'class="month">{{controller.date | luifMoment: \'MMM\' }}'+`

It will be easier to maintain.